drivers/oled/oled_driver.c

```diff
@@ -491,8 +491,9 @@ void oled_write_raw(const char *data, uint16_t size) {
     uint16_t cursor_start_index = oled_cursor - &oled_buffer[0];
     if ((size + cursor_start_index) > OLED_MATRIX_SIZE) size = OLED_MATRIX_SIZE - cursor_start_index;
     for (uint16_t i = cursor_start_index; i < cursor_start_index + size; i++) {
-        if (oled_buffer[i] == data[i]) continue;
-        oled_buffer[i] = data[i];
+        uint8_t c = *data++;
```

Comment:
Could you also do this for the ST7565 driver? (in a separate PR, of course, as it's currently in `develop`).

Reply:
Done in #13209.